### PR TITLE
implemente QueueDAO.containsMessage() for mysql and postgres. linked issue: 1892

### PR DESCRIPTION
**Pull request by Yingsheng-eroad**

### CODE REVIEW

**Comment by Yingsheng-eroad** [comment]
#1892

**Comment by kishorebanala** [comment]
Hey @mashurex @jvemugunta Can you help review these MySQL / Postgres module changes please?

File: mysql-persistence/src/test/java/com/netflix/conductor/dao/mysql/MySQLQueueDAOTest.java [hunk 1]
```diff
@@ -118,6 +119,32 @@ public void complexQueueTest() {
 		size = dao.getSize(queueName);
 		assertEquals(0, size);
 	}
+	@Test
```
Comment (demichej) [inline mysql-persistence/src/test/java/com/netflix/conductor/dao/mysql/MySQLQueueDAOTest.java:122]: can we put the `@Test` below the comment?

**Comment by demichej** [comment]
Looks good to me. Thanks for fixing for Mysql and Postgres.

File: mysql-persistence/src/test/java/com/netflix/conductor/dao/mysql/MySQLQueueDAOTest.java (hunk 1 above)
Comment (Yingsheng-eroad) [reply mysql-persistence/src/test/java/com/netflix/conductor/dao/mysql/MySQLQueueDAOTest.java]: > can we put the `@Test` below the comment?

updated.

**Comment by mashurex** [comment]
It looks good to me as well.

**Review by mashurex — APPROVED** [review]

**Comment by kishorebanala** [comment]
Thank you @Yingsheng-eroad @mashurex and @demichej

**Pull request merged**